test/e2e_node/density_test.go

```diff
@@ -528,7 +532,8 @@ func createBatchPodSequential(f *framework.Framework, pods []*v1.Pod) (time.Dura
 	e2eLags := make([]e2emetrics.PodLatencyData, 0)
 	for _, pod := range pods {
 		create := metav1.Now()
-		f.PodClient().CreateSync(pod)
+		p := f.PodClient().Create(pod)
+		framework.ExpectNoError(wait.PollImmediate(500*time.Millisecond, 2*time.Minute, podRunningAndReady(f, p.Name)))
```

Comment:
The default timeout is 5 minutes (https://github.com/kubernetes/kubernetes/blob/e2206cae9dacb397369980bf1c9e76d04234845b/test/e2e/framework/timeouts.go#L23) - we might want to reference that here, rather than reducing it - or at least document why it's shorter.

Reply:
there are already plenty of these helpers https://github.com/kubernetes/kubernetes/blob/f173d01c011c3574dea73a6fa3e20b0ab94531bb/test/e2e/framework/pod/wait.go#L365

I see that the e2e_node keep drifting from the e2e frameworks and is much more unstable, I think that there is more benefit on keep things aligned, also consolidating the timeouts as Danielle suggests

Reply:
The intention for my change is to deduce the polling interval for getting the pod status Running as soon as possible, because the test is for performance, so 2s > 500ms could be more punctual for status change

Reply:
okay, agree that we should keep the timeout as it is originally

Reply:
https://storage.googleapis.com/k8s-triage/index.html?test=create%20a%20sequence%20of%20pods%20latency#ae63e7365ad14fba9b1e
The test case still flakes.

I agree that 500ms may help if the percent 50 is expected to under 5s.
- 2s interval is too big for a density check which is 5-10s.
```
				podStartupLimits: e2emetrics.LatencyMetric{
					Perc50: 5000 * time.Millisecond,
					Perc90: 9000 * time.Millisecond,
					Perc99: 10000 * time.Millisecond,
				},
```

However, as Antonio pointed out, there is already helper func like `WaitTimeoutForPodReadyInNamespace`.
Can we reuse it or add a help func for this configurable interval wait in `wait.go` for this?

Reply:
okay, thanks